### PR TITLE
#1785 Test retrieving boolean value from result toMap()

### PR DESCRIPTION
**Pull request by Sandychuang8**
Test retrieving boolean value from result toMap()

### CODE REVIEW

File: android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/QueryTest.java [hunk 1]
```diff
@@ -1652,6 +1652,63 @@ public void testJoinWithArrayContains() throws Exception {
             Log.e(TAG, "RESULT: " + r.toMap());
     }
 
+    //https://github.com/couchbase/couchbase-lite-android/issues/1785
+    @Test
+    public void testesultToMapReturnBoolVlaues() throws Exception {
```
Comment (djpongh) [inline android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/QueryTest.java]: testResultToMapReturnBoolValues() ??

**Comment by Sandychuang8** [comment]
> Not sure why I am reviewing this?

I did not assign this to you.

**Comment by pasin** [comment]
@borrrden Could you please help reviewing this PR?

**Comment by borrrden** [comment]
I don't know what to review.  This seems like a problem that is isolated to a Java quirk right?

**Pull request merged**